### PR TITLE
MDP-198:Article page component

### PR DESCRIPTION
**Pull request by kolesarovakristina**

### CODE REVIEW

File: src/components/ArticlePage/Articles.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+`;
+export const Article = () => (
+  <BackgroundImage>
+    <div className="p-6">
+      {article.map(({ src, date, title, subtitle, paragraph }) => (
```
Comment (pggl) [inline src/components/ArticlePage/Articles.tsx]: why is article array? there will be only one article per page

Comment (pggl) [reply src/components/ArticlePage/Articles.tsx]: and what is inside should be Article component

File: src/components/ArticlePage/Articles.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <div key={title} className="bg-white">
+          <ImageHeader
+            src={src}
+            className="flex w-full h-64 justify-center m-0 p-3"
+          />
+          <Layout
```
Comment (pggl) [inline src/components/ArticlePage/Articles.tsx]: wrong name for component

File: src/components/Testing.tsx [hunk 3]
```diff
@@ -1,4 +1,9 @@
 import React, { FC } from "react";
 import { RouteComponentProps } from "@reach/router";
+import { Article } from "./ArticlePage/Articles";
 
-export const Testing: FC<RouteComponentProps> = () => <div />;
+export const Testing: FC<RouteComponentProps> = () => (
```
Comment (pggl) [inline src/components/Testing.tsx]: revert all this changes, it is only for development

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/static/svg/group-13.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1024" height="1253" viewBox="0 0 1024 1253">
```
Comment (pggl) [inline src/static/svg/group-13.svg]: wrong name for this svg, what is group-13.svg?

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/ArticleComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import backgroundImage from "../../static/svg/backgroundImage.svg";
+import { Article } from "./Article";
+import articleImage from "../../static/svg/articleImage.png";
+import { ImageHeader } from "../Header/ImageHeader";
+
+const BackgroundImage = styled.div`
```
Comment (garySiladi) [inline src/components/ArticlePage/ArticleComponent.tsx]: can you rename this to something like `ArticleWrapper`

**Review by garySiladi — CHANGES_REQUESTED** [review]
.

File: src/components/ArticlePage/ArticleWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import backgroundImage from "../../static/svg/backgroundImage.svg";
+import { Article } from "./Article";
+import articleImage from "../../static/svg/articleImage.png";
+import { ImageHeader } from "../Header/ImageHeader";
+
+const BackgroundImage = styled.div`
```
Comment (garySiladi) [inline src/components/ArticlePage/ArticleWrapper.tsx]: rename this to something like `ArticleContainer`

**Review by garySiladi — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/ArticleContainer.tsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import backgroundImage from "../../static/svg/backgroundImage.svg";
+import { Article } from "./Article";
+import articleImage from "../../static/svg/articleImage.png";
+import { ImageHeader } from "../Header/ImageHeader";
+
+const BackgroundImage = styled.div`
```
Comment (garySiladi) [inline src/components/ArticlePage/ArticleContainer.tsx]: you should import this as an external component. are you waiting on your landing page PR to be merged, and will reuse from there?

Comment (kolesarovakristina) [reply src/components/ArticlePage/ArticleContainer.tsx]: When landing page will be merged I change this.

**Review by garySiladi — DISMISSED** [review]
ok from my side just update with the component from MDP-185 once that's merged

File: src/components/ArticlePage/Article.tsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+  title,
+  subtitle,
+  paragraph,
+  source,
+  text
+}) => (
+  <div className="ml-6 mr-6 lg:ml-24">
+    <div className="lg:mr-24">
+      <div className="font-lato text-xs font-normal text-grey pt-4">{date}</div>
+      <div className="font-gilbert text-2xl font-bold text-black pb-4 pt-5">
```
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: if this is title there should be h1 tag

File: src/components/ArticlePage/Article.tsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+        {title}
+      </div>
+      <div className="font-lato text-base font-bold text-black pb-4">
```
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: and this should be h2 tag

File: src/components/ArticlePage/Article.tsx (hunk 8 above)
Comment (pggl) [reply src/components/ArticlePage/Article.tsx]: and there is also special article tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

File: src/components/ArticlePage/ArticleContainer.tsx [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  background-image: url(${backgroundImage});
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+`;
+export const ArticleContainer = () => {
+  const [articleData] = useState({
```
Comment (pggl) [inline src/components/ArticlePage/ArticleContainer.tsx]: we don't need useState

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/Article.tsx [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+export const Article: FC<IProps> = ({
+  date,
+  title,
+  subtitle,
+  paragraph,
+  source,
+  text
+}) => (
+  <article className="ml-6 mr-6 lg:ml-24">
+    <article className="lg:mr-24">
```
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: there should be only one article tag

**Review by pggl — CHANGES_REQUESTED** [review]

**Review by pggl — APPROVED** [review]

**Pull request merged**